src/NzbDrone.Core.Test/HealthCheck/Checks/AppDataLocationFixture.cs

```diff
@@ -9,6 +11,14 @@ namespace NzbDrone.Core.Test.HealthCheck.Checks
     [TestFixture]
     public class AppDataLocationFixture : CoreTest<AppDataLocationCheck>
     {
+        [SetUp]
+        public void Setup()
+        {
+            Mocker.GetMock<ILocalizationService>()
+                  .Setup(s => s.GetLocalizedString(It.IsAny<string>()))
+                  .Returns("Some Warning Message");
+        }
+
```

Comment:
Add this to the fixture of the failing test and it should start working again for you